### ISSUE TITLE
Unable to create experiment for kubeflow pipeline.

### ISSUE THREAD

**Issue by keshavpeswani** [kubeflow/kubeflow#5664]
/kind question

**Question:**
Hi Everyone,

I have been using following version of kubeflow pipleline sdk  :

```
kfp                                   0.1.29         
kfp-server-api                        0.1.18.3   
```

When i try to run the following from jupyter notebook started from kubeflow : 

```
client = kfp.Client()
aws_experiment = client.create_experiment(name='aws')
```

I get the following error : 

ApiException: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'content-type': 'application/json', 'trailer': 'Grpc-Trailer-Content-Type', 'date': 'Fri, 05 Mar 2021 12:22:58 GMT', 'x-envoy-upstream-service-time': '2', 'server': 'envoy', 'transfer-encoding': 'chunked'})
HTTP response body: {"error":"Validate experiment request failed.: Invalid input error: Invalid resource references for experiment. Expect one namespace type with owner relationship. Got: []","message":"Validate experiment request failed.: Invalid input error: Invalid resource references for experiment. Expect one namespace type with owner relationship. Got: []","code":3,"details":[{"@type":"type.googleapis.com/api.Error","error_message":"Invalid resource references for experiment. Expect one namespace type with owner relationship. Got: []","error_details":"Validate experiment request failed.: Invalid input error: Invalid resource references for experiment. Expect one namespace type with owner relationship. Got: []"}]}

Here jupyter is running in another namespace as opposed to kubeflow running in "kubeflow" namespace. What possibly is going wrong over here ? Can someone help me here ?

Thanks
Keshav

**Comment by DavidSpek** [comment]
@keshavpeswani You should be using the same version number for `kfp` and `kfp-server-api`. Also, is there a reason you are using such an old version? The current version is 1.4.1. Also, when trying to run a pipeline from a notebook you are probably running into https://github.com/kubeflow/pipelines/issues/4440. I believe the fix for this has been implemented and will be included in Kubeflow 1.3. 

/area jupyter

**Comment by keshavpeswani** [comment]
Thanks a lot @DavidSpek . Upgrading the kfp sdk and following the issue, solved the issue for me. Thanks once again.
